.github/workflows/ci.yml

```diff
@@ -26,6 +44,14 @@ jobs:
     # For pull requests it's not necessary to checkout the code
     - uses: "actions/checkout@v4"
 
+    - name: "Setup upterm session"
+      if: ${{ (true == inputs.enableUpterm) && (inputs.breakOnJob == github.job) }}
+      uses: owenthereal/action-upterm@v1
+      with:
+        limit-access-to-actor: true
+        limit-access-to-users: "bryanhonof,tomberek,dcarley,billlevine,ysndr,zmitchell,limeytexan,mkenigs,stahnma,garbas"
```

Comment:
can this use gh teams? i.e. the flox/cli team

Reply:
Sadly enough, no. I could make some changes to it though.